bucket_map/src/bucket.rs

```diff
@@ -266,10 +266,10 @@ impl<T: Clone + Copy> Bucket<T> {
                 let data_bucket = &self.data[elem.data_bucket_ix() as usize];
                 let loc = elem.data_loc(data_bucket);
                 //debug!(                    "DATA FREE {:?} {} {} {}",                    key, elem.data_location, data_bucket.capacity, elem_uid                );
-                data_bucket.free(loc, elem_uid).unwrap();
+                data_bucket.free(loc, elem_uid);
             }
             //debug!("INDEX FREE {:?} {}", key, elem_uid);
-            self.index.free(elem_ix, elem_uid).unwrap();
+            self.index.free(elem_ix, elem_uid);
```

Comment:
As you can see, all calls to `free()` immediately call `unwrap()`, so errors are never handled, we just panic.